### ISSUE TITLE
Templated slack messages

### ISSUE THREAD

**Issue by BHare1985** [PeteGoo/tcSlackBuildNotifier#119]
Currently our company uses the standard github notifications that run when a new pull request is submitted. Unfortunately that doesnt mean the build tests have fully completed yet so sometimes we review a pull request and have to wait on the tests to finish.

What I want to request is ability to notify slack of new request build passes. I know that can be done right now but the notifier only gives useful links to teamcity and not to Github. What would be nice is ability to get a link to the github pull request.

Generically speaking it would be awesome to have a templated message where could set the message ourselfs to be like  `Tests passed on Build %TeamCity.BuildName%, visit here %Github.Url%`

**Comment by PeteGoo** [comment]
I see, the way we handle this in our company is to use a label on the PR called `ready for review`. We then have an api that received webhooks from GitHub that posts to Slack when a PR is ready for review.

This means that the PR author is deciding when it is ready for review. I push loads of times to a PR branch before I'm ready for someone to look at it.

I did a talk on this, [the slides should give you an idea of what I mean](https://speakerdeck.com/petegoo/devops-learning-how-to-go-fast-without-tripping-over-the-cables?slide=14).

**Comment by pointerxr** [comment]
@BHare1985 does this help you or yo you still want this?

**Comment by pointerxr** [comment]
Closing in favor of #63
